sys/net/gnrc/doc.txt

```diff
@@ -0,0 +1,163 @@
+/**
+ * @defgroup    net_gnrc Generic (gnrc) network stack.
+ * @ingroup     net
+ * @brief       RIOT's modular default IP network stack.
+ *
+ * # About
+ *
+ * This module is currently the default network stack of RIOT and includes many components
+ * ranging from a simple MAC protocol (@ref net_gnrc_nomac) through a fully-featured IPv6
+ * implementation (@ref net_gnrc_ipv6) with 6LowPAN extensions (@ref net_gnrc_sixlowpan) to
+ * an UDP implementation (@ref net_gnrc_udp) and @ref net_gnrc_rpl.
```

Comment:
Might be beneficial to use the `@ref <module> "text"` syntax here too, to get rid of all the parenthesis